### PR TITLE
Group candidates by committee name

### PR DESCRIPTION
**Pull request by rwalek668**
Grouping candidates by committee name will allow the correct number of entries to be returned.

### CODE REVIEW

**Comment by alessandro-pianetta** [comment]
@geleazar1000111 If you have the correct permissions, could you just give this a quick look and approve or ping me to approve it if you can't? Just wanted to get someone more familiar than me to look at it before it gets merged.

**Review by geleazar1000111 — APPROVED** [review]

**Comment by geleazar1000111** [comment]
lgtm

**Pull request merged**
